src/depends.py

```diff
@@ -137,6 +137,7 @@ def detectOSRelease():
         for line in osRelease:
             if line.startswith("NAME="):
                 detectOS.result = OS_RELEASE.get(
+                    line.replace('"', '').split("=")[-1].strip().lower().split(" ")[0])
                     line.replace('"', '').split("=")[-1].strip().lower())
```

Comment:
duplicated here on merge conflict resolving. it'll take a bit longer on trying a selective mergeing workflow.